doc/page-properties.h

```diff
@@ -0,0 +1,129 @@
+/**********************************************************************
+**
+** Copyright (C) Luxoft Sweden AB 2020
+**
+** This file is part of the documentation of the FaceLift project
+**
+** This work is licensed under the Creative Commons
+** Attribution-ShareAlike 4.0 International License. To view a copy of
+** this license, visit http://creativecommons.org/licenses/by-sa/4.0/
+** or send a letter to Creative Commons, PO Box 1866, Mountain View,
+** CA 94042, USA.
+**
+** SPDX-License-Identifier: CC-BY-SA-4.0
+**
+**********************************************************************/
+
+/**
+\page page-properties Properties
+
+\section properties-intro Introduction
+Properties are the part of the interface definition described in *.qface file. Eventually they will be generated as C++ and QML properties of the interface. For more details about properties in *.qface files see documentation at https://github.com/Pelagicore/qface
```

Comment:
"Interface definition files (*.qface) can define properties. See https://github.com/Pelagicore/qface for information about QFace interface definition."